cmd/add.go

```diff
@@ -21,6 +27,10 @@ var addCmd = &cobra.Command{
 
 func init() {
 	RootCmd.AddCommand(addCmd)
+	addCmd.Flags().StringVarP(&organization, "organization", "", "", "Okta organization name")
+	addCmd.Flags().StringVarP(&oktaRegion, "okta-region", "", "", "Okta region (us, emea, preview)")
```

Comment:
What do you think about dropping `okta` from the flag? So it's just `--region`,`--domain`, `--username`?

Reply:
Yep sounds good, `region` could be confused with AWS region but if you don't think that's a problem I can just remove okta from all fields.